src/features/people/PeopleDetails/index.js

```diff
@@ -18,20 +25,94 @@ const PersonDetails = () => {
 
     const details = useSelector(selectDetails);
     const status = useSelector(selectStatus);
+    const actorCast = useSelector(selectPersonCast);
+    const actorCrew = useSelector(selectPersonCrew);
+    const [genres, setGenres] = useState([]);
+
+    useEffect(() => {
+        const fetchGenres = async () => {
+            const genresData = await getGenreList();
+            setGenres(genresData.genres);
+        };
+        fetchGenres();
+    }, []);
+
+    const nameGenres = (genreIds) => {
+        if (!genres || genres.length === 0) {
+            return [];
+        }
+
+        return genreIds.map((id) => {
+            const genre = genres.find((genre) => genre.id === id);
+            return genre ? genre.name : '';
+        });
+    };
+
+    const getReleaseYear = (releaseDate) => {
+        const date = new Date(releaseDate);
+        return date.getFullYear();
+    };
+
+    const formatVote = (vote) => vote.toLocaleString(
+        'pl-PL',
+        {
+            maximumFractionDigits: 1,
+        }
+    );
 
     return (
         status === "loading" ? <div>Loading...</div>
             : status === "error" ? <div>Error!</div>
                 : details ? (
-                    <>
-                        <PersonDetailsTile
-                            poster={details.profile_path}
+                    <Main>
+                        <DetailsTile
+                            posterPath={details.profile_path}
                             title={details.name}
-                            overview={details.biography}
-                            dateOfBirth={details.birthday}
-                            placeOfBirth={details.place_of_birth}
+                            description={details.biography}
```

Comment:
I think I saw people without bio (probably there are also obscure movies without descriptions) should we add some logic to DetailsTile to handle such situations and put some text there then? Otherwise there will be just empty space on the Tile.

What do You propose?

Reply:
Good point 😊 I agree, handling situations where there is no biography or description of the video is important for a refined user interface. In my next PR, I will add logic in the DetailsTile to display replacement text when a biography or description is missing, preventing blank spaces on the tile. My suggestion is:

`overview={details.biography || "No biography available"}`
`dateOfBirth={details.birthday}`
`placeOfBirth={details.place_of_birth}`
`description={details.biography || "No description available"}`

If you have any other suggestions for replacement text or other issues, please let me know.
Thanks for bringing this to my attention and attention to detail! 🚀👍🏼